src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -1,21 +1,56 @@
 import { Component } from '@angular/core';
 
+import { BsModalRef, BsModalService } from 'ngx-bootstrap';
+
 import { PoolService } from '../../../shared/api/pool.service';
+import { DeletionModalComponent } from '../../../shared/components/deletion-modal/deletion-modal.component';
+import { CdTableAction } from '../../../shared/models/cd-table-action';
 import { CdTableColumn } from '../../../shared/models/cd-table-column';
 import { CdTableFetchDataContext } from '../../../shared/models/cd-table-fetch-data-context';
 import { CdTableSelection } from '../../../shared/models/cd-table-selection';
+import { ExecutingTask } from '../../../shared/models/executing-task';
+import { FinishedTask } from '../../../shared/models/finished-task';
+import { Permission } from '../../../shared/models/permissions';
+import { AuthStorageService } from '../../../shared/services/auth-storage.service';
+import { TaskWrapperService } from '../../../shared/services/task-wrapper.service';
+import { Pool } from '../pool';
 
 @Component({
   selector: 'cd-pool-list',
   templateUrl: './pool-list.component.html',
   styleUrls: ['./pool-list.component.scss']
 })
 export class PoolListComponent {
-  pools = [];
+  pools: Pool[] = [];
   columns: CdTableColumn[];
   selection = new CdTableSelection();
+  modalRef: BsModalRef;
+  executingTasks: ExecutingTask[] = [];
+  permission: Permission;
+  tableActions: CdTableAction[];
 
-  constructor(private poolService: PoolService) {
+  constructor(
```

Comment:
You should subscribe to the `summaryService` and update the table each time it sends a new data.
ATM we don't see any information about running tasks related to pools.

Maybe we could extract and adapt the functions used in the `RBD List` component or simply copy them.

Reply:
Please have a look at #23555, it should help implement this.